src/xunit.performance.metrics/BenchmarkDurationEvaluator.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I was planning to split and re-arrange: attributes vs. metrics vs. evaluators, but it turned out to be a bigger task and I needed this change to go in first.